app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileScreen.kt

```diff
@@ -45,7 +45,7 @@ import com.wire.android.ui.theme.wireTypography
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun UserProfileScreen(viewModel: UserProfileViewModel) {
+fun UserProfileScreen(userId: String?, viewModel: UserProfileViewModel) {
```

Comment:
Looks like `userId` isn't used - is that intentional? Also, what should happen if `userId` is null? 🤔

Reply:
it will be used later on and it won't be `String` actually. And about null: minimum - crash, maximum -  some error message and closing the screen, I guess :)

Reply:
If this id is not a string, what would it be? an `Int`??